src/back/util/config.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file Configuration helper related methods.
+ */
+import config from 'config';
+
+/**
+ * Returns config to be used in the front-end
+ */
+export const frontEndConfig = () => {
+  const baseUrl = config.get('baseUrl');
+
+  return {
+    baseUrl,
+    graphqlEndpoint: `${baseUrl}graphql`,
```

Comment:
I'm not sure as to how the `graphqlEndpoint` variable is defined, but I'm guessing it's nature relates to `route/graphql`. In this case, are you entirely sure that `baseUrl` will always have the trailing `/`?

Reply:
Yes I can. I'm the one that defines baseUrl in the config file